scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,69 +1,107 @@
+    opt = options[i];
+
+    if (opt.selected) {
+    	if (opt.value !== "") {
+				chrom.push(opt.value);
+			}
     }
-    var emptyStart = document.createElement("option");
-    emptyStart.textContent = "";
-    emptyStart.value = "";
-    elem.appendChild(emptyStart); //Add an empty (blank) option to the select
   }
 
-  for(var i = 0; i < chrom_cytobands.length; i++) {
-    var opt = chrom_cytobands[i]
+	var chromPos = "";
+	if (typeof document.forms["filters_form"].elements["chrom_pos"] !== "undefined") {
+		chromPos = document.forms["filters_form"].elements["chrom_pos"].value;
+	}
 
-    // populate the cytoband start select
-    var interval = ["(start:",opt["start"], ")"].join("");
-    var optionText = [ chrom, opt["band"], interval ].join(" ");
+	var chromosome = "";
+	console.log("Populate cytobands")
+	var matchedChrName = chromPos.match(chromPosPattern)
+	if (chrom === [] && chromPos === "") {
+		startElem.value = "";
+		endElem.value = "";
+		return //only reset cytoband select element
+		// Set the selected chromosome name
+	} else if (chrom !== [] && matchedChrName !== null) {
+		chromosome = matchedChrName[1]
+	} else if (chrom === [] && matchedChrName !== null) {
+		chromosome = matchedChrName[1]
+	} else if (chrom == [] && matchedChrName == null) {
+		console.log("chrom is empty still")
+		return
+	} else if (typeof chrom === 'string' || chrom instanceof String) {
+		chromosome = chrom;
+	} else {
+		console.log("chrom is array: add multiple chr cytobands")
+		first = true
+		for (var i = 0; i < chrom.length; i++) {
+			if (chrom[i] === undefined) {
+				console.log("chrom " + i + " is undefined!")
+			}
+			console.log("chrom is array: add multiple chr cytobands. first " + i + ":" + chrom[i] + "("+ typeof (chrom[i])+")")
+			if (i !== 0) {
+				first = false
+			}
+			populateCytobandsSingleChr(cytobands, chrom[i], first)
+		}
+		return
+	}
+	populateCytobandsSingleChr(cytobands, chromosome, false)
+}
 
-    // populate the cytoband start select
-    var el = document.createElement("option");
-    el.textContent = optionText;
-    el.value = opt["start"];
-    if(startElem.value === el.value){
-      el.selected = true;
-    }
-    cytoStart.appendChild(el);
+function populateCytobandsSingleChr(cytobands, chromosome, first) {
+	console.log("cytobands 1: " + cytobands["1"] +  "("+ typeof (cytobands["1"])+")")
 
-    var interval = ["(end:",opt["stop"], ")"].join("")
-    var optionText = [ chrom, opt["band"], interval ].join(" ");
+	console.log("populate single chrom: " + chromosome + "("+ typeof (chromosome)+")")
+	var chrom_cytobands = cytobands[chromosome]["cytobands"]; // chromosome-specific cytobands
 
-    var el = document.createElement("option");
-    el.textContent = optionText;
-    el.value = opt["stop"];
-    if(endElem.value === el.value){
-      el.selected = true;
-    }
-    cytoEnd.appendChild(el);
-  }
+	for (elem of [cytoStart, cytoEnd]) {
+		if (first && elem.options.length > 0) {
```

Comment:
All this code is quite complicated, so yes, if somebody is interested in more than chromosome, I'd just deactivate the cytoband search, easy peasy!